src/components/NewComment.jsx

```diff
@@ -0,0 +1,56 @@
+import { useContext, useState } from "react";
+import { useParams } from "react-router-dom";
+import { UserContext } from "../contexts/UserContext";
+import { fetchUsers, postComment } from "../utils/api";
+import { matchUserImgs } from "../utils/user";
+import { TiTick } from "react-icons/ti";
+
+export default function NewComment({ setComments }) {
+  const { review_id } = useParams();
+  const { user } = useContext(UserContext);
+  const [newComment, setNewComment] = useState("");
+  const [posted, setPosted] = useState(false);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    Promise.all([postComment(review_id, newComment), fetchUsers()])
+      .then((result) => {
+        const avatarRef = matchUserImgs(result[1]);
+        result[0].avatar = avatarRef[result[0].author];
+        setComments((currComments) => {
+          return [...currComments, result[0]];
+        });
```

Comment:
Nice work updating the state ✅  though the latest comment is pushed to the bottom of the comment pile rather than the top, which made me think the state wasn't being updated until I scrolled down. To make this clearer for the user I would have the posted comment appear closest to the input box